scripts/download-sac-spec.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
```

Comment:
since this is user-facing we probably want it written in pure JS so our 3 windows users aren't in shambles